WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -158,7 +159,12 @@ private async Task<List<AliceClient>> ConfirmConnectionsAsync(IEnumerable<AliceC
 
 		private IEnumerable<Money> DecomposeAmounts(FeeRate feeRate)
 		{
-			return Coins.Select(c => c.Amount - feeRate.GetFee(c.ScriptPubKey.EstimateInputVsize()));
+			var allDenominations = BaseDenominationGenerator.Generate();
+			GreedyDecomposer greedyDecomposer = new(allDenominations);
+			var coins = Coins.Select(c => c.Amount - feeRate.GetFee(c.ScriptPubKey.EstimateInputVsize()));
+			var denominations = greedyDecomposer.Decompose(coins.Sum());
+
+			return coins; //TODO: return denominations instead
```

Comment:
what does the comment mean? the current return type of `IEnumerable<Money>` seems to do that already